blog/vr-volume-viewer.md

```diff
@@ -18,17 +23,20 @@ CCV has worked in scientific visualization since its inception back in 2003, col
 It is our objective to do research on the best tools to implement these types of applications, going from open source libraries to free to use 3D visualization tools such as blender, paraview and Unity 3D. We also follow software engineering standards to ensure sustainable maintenance and produce efficient and effective products.
 Historically, we have worked in the OpenGL rendering pipeline to place these datasets into 3D scenes. This library has been fully optimized for triangle mesh indexing and rendering 3D content for interactive real time simulations. Most of the time the programmer only needs to focus on implementing tools to apply transformation to those triangles and facilitate the manipulation of objects by the user.
 
-![image1](/content/images/blog/vr-volume-viewer/image1.png)  3D Bunny mesh represented as a set of triangles
```

Comment:
None of Camilo's images are showing up on my end, still. Jordan's blog's images work, though.

Reply:
thats weird. i would expect all images to not show up in the preview url above because of the `/content` preceding the image paths. we have to do this for now because the content repo is a submodule within the main `ccv-website-nuxt` repo.